src/components/DocumentConfigurationTab.tsx

```
@@ -0,0 +1,351 @@
+	Flex,
+} from '@chakra-ui/react';
+import { AddIcon, DeleteIcon } from '@chakra-ui/icons';
+import { updateMapping, getMapping } from '../services/admin/admin';
+
+const DocumentConfigurationTab = () => {
+	const toast = useToast();
+
+	// --- State for document configurations and errors ---
+	const [documentConfigs, setDocumentConfigs] = useState([]); // List of document configurations
+	const [errors, setErrors] = useState({}); // Validation errors
+	const [loading, setLoading] = useState(false); // Loading state
```

ESLint

[error] 'loading' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)